### PR TITLE
Wrap error messages in div with role status

### PR DESCRIPTION
**Pull request by kiruys**
- File input
- Text area
- Form field
- error in KTO form

### CODE REVIEW

File: src/components/TextArea/TextArea.test.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import userEvent from '@testing-library/user-event'
 
 import { withAppContext } from 'test/utils'
 
-import TextArea from '..'
+import TextArea from './index'
```
Comment (janjaap) [inline src/components/TextArea/TextArea.test.tsx:8]: Linter should've complained about this. Should be `import TextArea from '.'`

File: src/signals/incident/components/form/FormField/index.test.tsx [hunk 2]
```diff
@@ -5,10 +5,10 @@ import { Validators } from 'react-reactive-form'
 
 import { withAppContext } from 'test/utils'
 import type { FormMeta } from 'types/reactive-form'
-import { createRequired } from '../../../../services/custom-validators'
+import { createRequired } from '../../../services/custom-validators'
 
-import type { FormFieldProps } from '../FormField'
-import FormField from '..'
+import type { FormFieldProps } from './FormField'
+import FormField from './index'
```
Comment (janjaap) [inline src/signals/incident/components/form/FormField/index.test.tsx:11]: Should be `import FormField from '.'`

**Review by janjaap — APPROVED** [review]
A couple of remarks; minor issues

File: src/components/TextArea/TextArea.test.tsx (hunk 1 above)
Comment (kiruys) [reply src/components/TextArea/TextArea.test.tsx]: fixed

**Pull request merged**